robot/EDA/resources/ContactsPageObject.py

```diff
@@ -91,17 +110,18 @@ def select_contact(self,contact_firstname, contact_lastname):
     def validate_preferred_phone_form(self):
         """ Test the preferred phone fields and functionality """
 
-        self.selenium.wait_until_page_contains_element(
-            contacts_locators["edit_contact"], 
-            timeout=60,
-            error="Edit button not found for Contact"
-        )
-        self.open_item(
-            contacts_locators["edit_contact"], 
-            "Edit button not found for Contact", 
-            False
-        )
-
+#         self.selenium.wait_until_page_contains_element(
+#             contacts_locators["edit_contact"], 
+#             timeout=60,
+#             error="Edit button not found for Contact"
+#         )
+#         self.open_item(
+#             contacts_locators["edit_contact"], 
+#             "Edit button not found for Contact", 
+#             False
+#         )
+        self.salesforce.click_object_button("Edit")
+        self.salesforce.wait_until_modal_is_open()
         self.selenium.driver.execute_script(
             "arguments[0].scrollIntoView()", 
             self.selenium.driver.find_element_by_xpath(contacts_locators["preferred_phone"])
```

Comment:
consider assigning to a variable and calling the variable?

Reply:
Not making much changes to existing code (only simple needed stuff)as it will all get rewritten later